src/SpecDataHolder.sol

```diff
@@ -32,14 +32,41 @@ contract SpecDataHolder is
     _disableInitializers();
   }
 
+  function initialize(address _raftAddress, address _nextOwner)
```

Comment:
It is unclear to me why we are removing this? Would this not render it "unupgradable"?